be/src/runtime/memory/chunk_allocator.cpp

```diff
@@ -219,6 +224,10 @@ void ChunkAllocator::free(const Chunk& chunk, MemTracker* tracker) {
         }
     } while (!_reserved_bytes.compare_exchange_weak(old_reserved_bytes, new_reserved_bytes));
 
+    // Reduce set metric frequency
+    if (_reserved_bytes % 100 == 32) {
```

Comment:
How to make sure the correctness?

At the first look, (_reserved_bytes % 100 < 32) or (_reserved_bytes % 100 > 32) both will not update the metric.

Reply:
At the first look, ChunkAllocator will allocate/free many times, the memory size of each allocate/free is a multiple of 2, so `_reserved_bytes% 100 == 32` will definitely happen, and the latest `_reserved_bytes` value will be set each time .

The real-time and accurate `_reserved_bytes` value is not required. Usually, the value of `_reserved_bytes` is equal to ChunkAllocator MemTracker. The `_reserved_bytes` metric is only concerned when verifying the accuracy of MemTracker.

Therefore, reduce the number of sets and reduce the performance impact.